### DISCUSSION TITLE
FR: 'Editor' of entries

### DISCUSSION THREAD

**Discussion by mikesnoeren** [craftcms/cms#8758] [Ideas]
Currently craft doesn't show enough information for who makes and edits an entry:

We have the following information available when we make an entry:
- Date created
- Author

We have the following information available when we update an entry:
- Date updated

Why not add 'Editor' so we can see who edited an entry? We have had multiple situations where it would be useful to be able to see this information. This could be added to all types, not just to entries.

**Comment by brandonkelly** [comment]
If the entry’s section has versioning enabled, you can tell who last edited the entry by checking its latest revision:

```twig
{% set lastRevision = craft.entries()
    .revisionOf(entry)
    .one() %}

{% set lastEditor = lastRevision.creator ?? null %}

{% if lastEditor %}
    <p>Last edited by {{ lastEditor.name }} on {{ lastRevision.dateCreated|date('short') }}</p>
{% endif %}
```

**Comment by mikesnoeren** [comment]
I am aware of the possibility to get the Editor on template level. Unfortunately that doesn't fit out needs. It would be nice to see the Editor within Craft itself. So you could sort entries on last editor for example.
